GAE/src/com/gallatinsystems/survey/dao/QuestionDao.java

```diff
@@ -593,8 +593,8 @@ public Question getByKey(Long questionId) {
                     if (Question.Type.OPTION == q.getType()) {
                         q.setQuestionOptionMap(optionDao.listOptionByQuestion(q.getKey().getId()));
                     }
-                    q.setTranslationMap(translationDao.findTranslations(
-                            ParentType.QUESTION_TEXT, q.getKey().getId()));
+                    q.setTranslations(translationDao.findTranslations(
+                            q.getKey().getId(), ParentType.QUESTION_TEXT));
```

Comment:
```suggestion
                            q.getKey().getId(), ParentType.QUESTION_TEXT, ParentType.QUESTION_TIP));
```

I believe in this one indeed we need to add the Question top as well.